utils/build-presets.ini

```diff
@@ -933,6 +933,71 @@ mixin-preset=buildbot_linux_crosscompile_android,tools=RA,stdlib=RD,build
 
 android-arch=aarch64
 
+# Preset for Fedora Linux
+[preset: buildbot_linux_fedora,no_test]
+mixin-preset=
+    mixin_lightweight_assertions,no-stdlib-asserts
+    mixin_linux_install_components_with_clang
+    mixin_buildbot_linux,no_test
+build-subdir=buildbot_linux
+
+lldb
+release
+test
```

Comment:
If you don't want to test, why add all these test flags?

Reply:
They're present in the `buildbot_linux` preset, which is included in `buildbot_linux,no_test`. I figured to make as few changes as needed to work with Fedora so I only removed the ones that I already remove as a pre-build patch.

Reply:
I tried that, but the build script didn't understand `--skip-build-ninja` and I also tried setting `build-ninja=0` and the end result is that the build segfaults 100% of the time on both x86_64 and aarch64. I agree there's some redundancy in the patch; if there's a way to turn off a particular component, I haven't figured out how to do it properly. :\

Reply:
Can you try removing these 7 test flags and the `no_test` mixin above, then running that to make sure it works? I believe they just cancel each other out and do nothing.

Reply:
Okay, I modified the configuration and tested it; it built successfully and the resulting RPM works fine. I figure I should close this PR and open a new one with the modifications, right? BTW, here is a gist of what I used: https://gist.github.com/tachoknight/4f0f07a90ea6c40d4d44f750def73d31

Reply:
No, you can just apply your changes to this branch and commit them with `git commit --amend utils/`, then force push this branch with `git push -f origin main`, where `origin` may vary based on how you named your remote repos.

Btw, I tried overriding `--build-ninja` and passing `--build-ninja=0` and `--build-ninja=False` after it both worked. Maybe you got the order wrong, or presets act weird somehow.